src/Symfony/Component/Form/Form.php

```diff
@@ -1044,11 +1044,29 @@ public function createView(FormView $parent = null)
             $view->children[$name] = $child->createView($view);
         }
 
+        $this->sort($view->children);
```

Comment:
should this be done before or after `finishView` ?

Reply:
From my point of view, this should be done before `finishView()`.
 1) This is the final step of the view creation, thus I don't expect any view change after that.
 2) In case you want to change the current order to anything else (custom conditions) you could do it in `finishView()`.

Reply:
Indeed, if you have a type extension doing this currently (sorting fields on `finishView` by a custom criteria), we will break that chance if we sort after `finishView`.

Reply:
makes sense